test/models/trail_test.rb

```diff
@@ -1,5 +1,48 @@
 require "test_helper"
 
 class TrailTest < ActiveSupport::TestCase
+  def setup
+    trail1 = Trail.create(name: "howdy hike",
```

Comment:
I agree with Hound here...there is no need for the assignments to `trail1`, `trail2`, and `trail3`.